components/pages/workPageLayout/WorkPageLayout.tsx

```diff
@@ -1,19 +1,57 @@
 "use client"
 
 import { useQuery } from "@tanstack/react-query"
-import React from "react"
+import React, { useState } from "react"
 
+import { Button } from "@/components/shared/button/Button"
+import Player from "@/components/shared/publizonPlayer/PublizonPlayer"
+import ResponsiveDialog from "@/components/shared/responsiveDialog/ResponsiveDialog"
+import SmartLink from "@/components/shared/smartLink/SmartLink"
 import { useGetMaterialQuery } from "@/lib/graphql/generated/fbi/graphql"
+import { resolveUrl } from "@/lib/helpers/helper.routes"
 
 function WorkPageLayout({ wid }: { wid: string }) {
-  const data = useQuery({
+  const [isPlayerOpen, setIsPlayerOpen] = useState(false)
+
+  const { data } = useQuery({
     queryKey: useGetMaterialQuery.getKey({ wid }),
     queryFn: useGetMaterialQuery.fetcher({ wid }),
   })
 
+  const manifestations = data?.work?.manifestations.all
+
+  const identifier = manifestations?.[0].identifiers?.[0].value || ""
+
```

Comment:
What happens if we do not get data or work and therefore do not have manifestations? How is the rest of the code reacting?

Reply:
TODO is added here